src/config/config.ts

```diff
@@ -146,6 +160,13 @@ const config: AppIds = {
       ru: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus-ru',
       ar: 'https://govextra.gov.il/ministry-of-health/corona/corona-virus-ar'
     },
+    downloadMohLandingPage: {
+      he: 'https://govextra.gov.il/ministry-of-health/hamagen-app/download-he/',
+      en: 'https://govextra.gov.il/ministry-of-health/hamagen-app/download-en/',
+      ru: 'https://govextra.gov.il/ministry-of-health/hamagen-app-ru/download-ru/',
+      ar: 'https://govextra.gov.il/ministry-of-health/hamagen-app/download-ar/',
+      am: 'https://govextra.gov.il/ministry-of-health/hamagen-app/download-he/',
```

Comment:
Should be download-am

Reply:
Understood,But please change to Amharic download url please(it is currently a Hebrew one)